npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -531,6 +532,11 @@ def handle_npm_updater_error(error)
             raise Dependabot::DependencyFileNotResolvable, msg
           end
 
+          if error_message.match(NESTED_ALIAS)
+            msg = "nested aliases not supported"
```

Comment:
This is probably because they are using an older version of npm; nested aliases are supported in version npm 6.9+ (https://stackoverflow.com/questions/54085943/npm-err-invalid-dependency-type-requested-alias). Can we update the message to give the user a hint of upgrading their npm version?

Also tagging @jonjanego as FYI as we've discussed similar themes (e.g. auto-install package manager versions etc.)

Reply:
@abdulapopoola you are right, How about "_Nested aliases are not supported with NPM version <6.9.0_"

Reply:
Thanks, yeah that makes sense

Reply:
Maybe this framing is clearer? ""Nested aliases are not supported in NPM versions earlier than 6.9.0.""

Reply:
@abdulapopoola , sounds better. ill change the message